libkbfs/crypto_common.go

```diff
@@ -546,3 +546,37 @@ func (c CryptoCommon) DecryptMerkleLeaf(encryptedLeaf EncryptedMerkleLeaf, privK
 	}
 	return &leaf, nil
 }
+
+// EncryptTLFCryptKeys implements the Crypto interface for CryptoCommon.
+func (c CryptoCommon) EncryptTLFCryptKeys(oldKeys []TLFCryptKey, key TLFCryptKey) (
+	encryptedKeys EncryptedTLFCryptKeys, err error) {
+	encodedKeys, err := c.codec.Encode(oldKeys)
+	if err != nil {
+		return
+	}
+
+	encryptedData, err := c.encryptData(encodedKeys, key.data)
+	if err != nil {
+		return
+	}
+
+	encryptedKeys = EncryptedTLFCryptKeys(encryptedData)
+	return
+}
+
+// DecryptTLFCryptKeys implements the Crypto interface for CryptoCommon.
+func (c CryptoCommon) DecryptTLFCryptKeys(encKeys EncryptedTLFCryptKeys, numKeys int, key TLFCryptKey) (
+	[]TLFCryptKey, error) {
+	encodedKeys, err := c.decryptData(encryptedData(encKeys), key.data)
+	if err != nil {
+		return nil, err
+	}
+
+	oldKeys := make([]TLFCryptKey, numKeys)
```

Comment:
I don't think you actually need to pass in `numKeys`, do you? https://github.com/msgpack/msgpack/blob/master/spec.md#formats-array says that the array length is encoded. You could probably just go `var oldKeys []TLFCryptKey`.